Tasks/NodeToolV0/nodetool.ts

```diff
@@ -63,7 +64,12 @@ async function getNode(versionSpec: string, checkLatest: boolean) {
             // query nodejs.org for a matching version
             version = await queryLatestMatch(versionSpec);
             if (!version) {
-                throw new Error(`Unable to find Node version '${versionSpec}' for platform ${osPlat} and architecture ${osArch}.`);
+                // If we can't find it from nodejs.org, try iojs.org (has versions 1-3)
```

Comment:
Add an extra condition to also check the version we are looking for and make sure it's in the range of io? Don't want to look for later versions that can't be found.